worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -220,22 +208,31 @@ public void save(Actor actor, LocalSession session,
     public void share(Actor actor, LocalSession session,
                       @Arg(desc = "Schematic name. Defaults to name-millis", def = "")
                           String schematicName,
-                      @Arg(desc = "Format name.", def = "sponge")
-                          String formatName) throws WorldEditException {
+                      @Arg(desc = "Share location", def = "enginehub")
```

Comment:
this doesn't strictly match a destination, should probably be "eh_paste" or whatever